### DISCUSSION TITLE
Running a vpncloud node inside a container

### DISCUSSION THREAD

**Discussion by sandreae** [dswd/vpncloud#213] [Q&A] [answered]
Hello,

I wondered if there are any obvious downsides to running a VPNcloud node inside a Docker container? My reason for wanting to do this rather than running it as a service is mainly in order to use my regular deployment platform (dokku) and not run extra services on this dedicated server. The node I want to run here would be the "main" node which is reliably online at a static IP for a number of other dynamic peers to connect via.

Many thanks @dswd for all your work on VPNcloud!

**Comment by dswd** [comment, marked as answer]
Hi there,

so there is one downside. In order to use virtual network interfaces inside containers, you need to add privileges to the container: net_admin

**Reply by sandreae** [reply]
Thanks for your reply, I have a docker repo [here](https://github.com/sandreae/vpncloud-docker) if it's useful for anyone else as a reference.
